samples/dotnetsay/dotnetsay.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <Description>A simple .NET Core tool called "dotnetsay".</Description>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <LangVersion>9</LangVersion>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
```

Comment:
Can `LangVersion` just be removed?

Reply:
That would mean that targeting .NET 6 would give you C# 10, whereas latest will give you C# 11 or 12 depending on whether you have a 7 or 8 SDK.

Given the purpose of these samples is an intro to .NET things, it's nice to have "latest" in case they want to try some new C# feature they've read about. Where a language feature requires some library/runtime stuff that's missing, it should give you a nice error -- confirmed with Jared..